.env.example

```
@@ -0,0 +1,42 @@
+# KAFKA_BOOTSTRAP_SERVERS=<KAFKA_INTERNAL_IP>:9092
+# EUREKA_DEFAULT_ZONE=http://<EUREKA_VM_1_INTERNAL_IP>:9001/eureka/,http://<EUREKA_VM_2_INTERNAL_IP>:9002/eureka/
+# MONITORING_HOST=<MONITORING_VM_EXTERNAL_IP>
+
+# ── prod 환경 예시 ───────────────────────────────────────────────────
+# SPRING_PROFILES_ACTIVE=prod
+# CONFIG_SERVER_URL=http://<CONFIG_SERVER_HOST>:9000
+# REDIS_SENTINEL_1=<REDIS_1_INTERNAL_IP>:26379
+# REDIS_SENTINEL_2=<REDIS_2_INTERNAL_IP>:26379
+# REDIS_SENTINEL_3=<REDIS_3_INTERNAL_IP>:26379
+# REDIS_PASSWORD=****
+# CONFIG_SERVER_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)